extension/src/popup/components/sendPayment/SendConfirm/TransactionDetails/index.tsx

```diff
@@ -34,13 +34,7 @@ import { AccountAssets } from "popup/components/account/AccountAssets";
 
 import "./styles.scss";
 
-export const TransactionDetails = ({
```

Comment:
transaction details "back" should go to accountHistory
https://stellarorg.atlassian.net/browse/WAL-128?filter=10043